src/App.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
-
 export const App: React.FC = () => {
+  const totalItems = 42;
+  const firstPage = 1;
```

Comment:
You can save these values outside the component to not create them on each render